tests/core/test_scheduler.py

```diff
@@ -10,6 +10,12 @@
 from .utils import create_dummy_prompt
 
 
+def get_sequence_groups(scheduler_output):
+    return [
+        seq_group for seq_group, _ in scheduler_output.scheduled_seq_groups
```

Comment:
since you are reuturning named tuple, just access by name instead? 
```suggestion
        s.seq_group for s in scheduler_output.scheduled_seq_groups
```